src/OpenTelemetry/CHANGELOG.md

```diff
@@ -70,6 +70,12 @@
   Specification](https://github.com/open-telemetry/opentelemetry-specification/pull/3820).
   ([#5404](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5404))
 
+* **Experimental (pre-release builds only):** The `ExemplarFilter` used by SDK
+  `MeterProvider`s can now be controlled via the `OTEL_METRICS_EXEMPLAR_FILTER`
+  environment variable. For details see: [OpenTelemetry Environment Variable
```

Comment:
nit: i personally prefer if we list the 3 supported values right in this changelog.

Reply:
particularly since env. vars are optional, and OTel .NET don't support all of them, so its easier if we list it without having to send ppl to spec repo.

Reply:
Updated